scripts/Pdf2Txt.py

```diff
@@ -0,0 +1,29 @@
+import os
+
+import pdftotext
+
+from scripts.task import Task
+
+
+class Pdf2Txt(Task):
+
+    def __init__(self, input_dir, output_dir):
+        super().__init__()
+        self.output_dir = output_dir
+        self.input_dir = input_dir
+
+    def execute(self):
+        for filename in os.listdir(self.input_dir):
+            if filename.endswith(".pdf"):
+                ofile = filename.split(".")[0] + ".txt"
```

Comment:
`filename.replace('pdf', 'text')` ?

Reply:
Don't want to fail for cases where pdf in present in the actual filename